File: 프로그래머스/Lv2/숫자의 표현.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 1;
    int temp=0;
    
    for(int i=1;i<=n;i++){
        temp=i;
        for(int j=i+1;j<=n;j++){
            if(temp+j>n) break;
            else if(temp+j==n){
                answer++;
                break;
            }else{
                temp+=j;
            }
        }
    }
    
    return answer;
}

File: 프로그래머스/Lv2/숫자의 표현.py
def solution(n):
    answer = 1
    
    for i in range(1,int(n/2)+1):
        sum=0
        for j in range(i,n):
            sum+=j
            if(sum>=n):
                if(sum==n): answer+=1
                break
            
    
    return answer
